NOTICES.md

```diff
@@ -0,0 +1,80 @@
+# Implemented notices
+
+Notices are declared in the [`Notice` module](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice).  Below are details of currently implemented notices.
+
+### Table of Errors
+
+| Error ID      | Error Title         |
+|---------------|---------------------------|
+| [E001](#E001) | | 
+| [E002](#E002) | | 
+| [E003](#E003) | | 
+| [E004](#E004) | | 
+| [E006](#E006) | | 
+| [E009](#E009) | | 
+| [E010](#E010) | | 
+| [E011](#E011) | | 
+| [E012](#E012) | |
+| [E013](#E013) | |
+| [E015](#E015) | |
+| [E016](#E016) | | 
+| [E017](#E017) | |
+| [E018](#E018) | | 
+| [E019](#E019) | | 
+| [E020](#E020) | | 
+| [E021](#E021) | | 
+| [E022](#E022) | Invalid Language Code | 
+| [E023](#E023) | Invalid Email | 
+
+### Table of Warnings
+
+| Warning ID    | Warning Title             |
+|---------------|---------------------------|
+| [W001](#W001) | | 
+| [W002](#W002) | | 
+| [W003](#W003) | | 
+| [W004](#W004) | | 
+| [W005](#W005) | Route description equals Route name | 
+| [W006](#W006) | Route Color Contrast | 
```

Comment:
All rule names should be an example of the bad data - so this should be something like "Insufficient route color contrast" or "Not enough route color contrast".

Also, we should be consistent in the capitalization of the words within the rule titles. I find that normal sentence case is easiest to read - like "Insufficient route color contrast".